src/pipeline-options.hpp

```diff
@@ -32,16 +32,17 @@ LOST_CLI_OPTION("max-mismatch-probability" , float      , maxMismatchProb
 LOST_CLI_OPTION("attitude-algo"            , std::string, attitudeAlgo           , ""  , optarg       , "dqm")
 
 // OUTPUT COMPARISON
-LOST_CLI_OPTION("centroid-compare-threshold", float      , centroidCompareThreshold, 1 , atof(optarg), kNoDefaultArgument)
-LOST_CLI_OPTION("attitude-compare-threshold", float      , attitudeCompareThreshold, 1 , atof(optarg), kNoDefaultArgument)
-LOST_CLI_OPTION("plot-raw-input"            , std::string, plotRawInput            , "", optarg      , "-")
-LOST_CLI_OPTION("plot-input"                , std::string, plotInput               , "", optarg      , "-")
-LOST_CLI_OPTION("plot-output"               , std::string, plotOutput              , "", optarg      , "-")
-LOST_CLI_OPTION("print-centroids"           , std::string, printCentroids          , "", optarg      , "-")
-LOST_CLI_OPTION("print-attitude"            , std::string, printAttitude           , "", optarg      , "-")
-LOST_CLI_OPTION("compare-centroids"         , std::string, compareCentroids        , "", optarg      , "-")
-LOST_CLI_OPTION("compare-star-ids"          , std::string, compareStarIds          , "", optarg      , "-")
-LOST_CLI_OPTION("compare-attitudes"         , std::string, compareAttitudes        , "", optarg      , "-")
+LOST_CLI_OPTION("centroid-compare-threshold", float      , centroidCompareThreshold, 1   , atof(optarg)   , kNoDefaultArgument)
+LOST_CLI_OPTION("attitude-compare-threshold", float      , attitudeCompareThreshold, 1   , atof(optarg)   , kNoDefaultArgument)
+LOST_CLI_OPTION("plot-raw-input"            , std::string, plotRawInput            , ""  , optarg         , "-")
+LOST_CLI_OPTION("plot-input"                , std::string, plotInput               , ""  , optarg         , "-")
+LOST_CLI_OPTION("plot-output"               , std::string, plotOutput              , ""  , optarg         , "-")
+LOST_CLI_OPTION("print-centroids"           , std::string, printCentroids          , ""  , optarg         , "-")
+LOST_CLI_OPTION("print-attitude"            , std::string, printAttitude           , ""  , optarg         , "-")
+LOST_CLI_OPTION("compare-centroids"         , std::string, compareCentroids        , ""  , optarg         , "-")
+LOST_CLI_OPTION("compare-star-ids"          , std::string, compareStarIds          , ""  , optarg         , "-")
+LOST_CLI_OPTION("compare-attitudes"         , std::string, compareAttitudes        , ""  , optarg         , "-")
+LOST_CLI_OPTION("coverage-test"             , std::string, coverageTest            , ""  , optarg         , "-")
```

Comment:
It seems like you never actually use the string argument? If you don't, change it to a bool instead.